pdns/dnsdist-lua-rules.cc

```diff
@@ -440,6 +440,12 @@ void setupLuaRules()
       return std::shared_ptr<TimedIPSetRule>(new TimedIPSetRule());
     });
 
+  g_lua.writeFunction("PoolAvailableRule", [](std::string poolname) {
+    setLuaSideEffect();
```

Comment:
The `setLuaSideEffect()` line is not needed.

Reply:
Removed